src/main/controllers/UserResultsController.ts

```diff
@@ -73,6 +79,16 @@ export class UserResultsController extends RootController {
     });
   }
 
+  private processUIRoles(user: User): string {
+    let uiRoles = '';
+    if(user.roles) {
+      user.roles.forEach((element => {
+        uiRoles = uiRoles + element + '<br>';
+      }));
+    }
+    return uiRoles;
+  }
+
```

Comment:
I'm pretty sure what you're trying to achieve here can be done directly within the nunchucks template, it's better to leave the view logic within the view itself so then we can much easier verify that the controller is passing the correct data to the view (without it being pre-formatted for the view's sake)

Reply:
What can i use?
I saw there was something to add br tags automatically but it required there to be \n delimiters instead of commas so i tried doing a replace commas with \n then pushing that into the front end and using nl2br but neither seemed to exist. I just kept getting errors if i used it
found it here
https://mozilla.github.io/nunjucks/templating.html#nl2br

Reply:
should be fixed now.